src/cli/serve.jl

```diff
@@ -45,7 +45,7 @@ function setup_socket(file)
         """)
     # Include the basics
     write(io, "<script>", Patchwork.js_runtime(), "</script>")
-    write(io, """<script src="/assets/bower_components/webcomponentsjs/webcomponents.min.js"></script>""")
+    write(io, """<script src="/pkg/Escher/assets/bower_components/webcomponentsjs/webcomponents.min.js"></script>""")
```

Comment:
You're right. I wasn't really sure how https://github.com/shashi/Escher.jl/pull/74 implemented things.

I will change this. Again.